webrtc.html

```diff
@@ -895,6 +899,20 @@
                     <var>connection</var>.</p>
                   </li>
                   <li>
+                    <p>If <var>description</var> is set as a local description,
+                    and it creates a new SCTP association, as defined in
+                    [[!SCTP-SDP]], set the value of <var>connection</var>'s
+                    [[<code><a>sctpTransport</a></code>]] internal slot
+                    to a newly created <code><a>RTCSctpTransport</a></code>.</p>
```

Comment:
> Is it possible to initiate a new SCTP association before setRemoteDescription is called?

Yeah, "initiate" doesn't quite work either, especially given that SCTP uses an "INIT" message.

I'd like to not say something as vague as "If the description is associated with a new SCTP association"... How about "If the description *will lead to* a new SCTP association being established?"

Another option would be to only create a new object when applying an answer or pr-answer. The more I think about it, the more that seems like the more correct thing to do. For one, how is "maxMessageSize" set without any remote description?